test/integration/api/authentication.spec.js

```diff
@@ -1077,6 +1077,14 @@ test.serial('performs an OpenID authentication', async (t) => {
 })
 
 if (shouldExecuteBuiltInSSOTest) {
+  // Please note Github may apply a verification code at login when a new detected device
+  // even if 2-factor authentication is disabled
+  // https://github.blog/changelog/2019-07-01-verified-devices
+  // https://github.community/t/disable-remove-email-device-verification-prompt-on-login-not-the-2fa/2333
+
+  // You should login to Github with your test account credentials (username, password) before running
+  // this test in local environment
```

Comment:
Do you mean in a specific browser (given that tests OAuth2 tests run in Puppeteer)?

Reply:
I didn't encounter issues after sign in with Chromium. I suppose the new device verification detects a mix of user agent and ip address.
For local environment, signing in with any browser should pass the verification step, so testing puppeteer will be permitted.